cwl/sphinx/runcmd.py

```diff
@@ -0,0 +1,185 @@
+import csv
+import os
+import re
+import shlex
+import subprocess
+import sys
+
+from pathlib import Path
+
+from docutils.parsers.rst import directives
+from sphinx.directives import code
+
+""""
+Patched version of https://github.com/sphinx-contrib/sphinxcontrib-runcmd
+with default values to avoid having to re-type in every page. Also
+prepends commands with a value (``$``), see https://github.com/invenia/sphinxcontrib-runcmd/issues/1.
+Finally, it also checks if the command is ``cwltool``, and if then
+tries to remove any paths from the command-line (not the logs).
+"""
+
+__version__ = "0.2.0"
+
+# CONSTANTS
+RE_SPLIT = re.compile(r"(?P<pattern>.*)(?<!\\)/(?P<replacement>.*)")
+
+
+# These classes were in the .util module of the original directive.
+# TODO: PATCHED
```

Comment:
Can we get issues for the TODOs? I don't know what the request is here

Reply:
Ah, these TODO's are more a note to myself about what was patched :sweat_smile: Should I change it to `NOTE`? Sorry!

Reply:
Fixed. Removed the TODO markers, leaving the comments only.